common/src/main/java/com/frostwire/mp3/ID3v2CommentFrameData.java

```diff
@@ -106,9 +106,8 @@ public boolean equals(Object obj) {
         } else if (other.description == null) return false;
         else if (!description.equals(other.description)) return false;
         if (comment == null) {
-            if (other.comment != null) return false;
+            return other.comment == null;
         } else if (other.comment == null) return false;
-        else if (!comment.equals(other.comment)) return false;
-        return true;
+        else return comment.equals(other.comment);
```

Comment:
I'm not so sure this is logically equivalent, that return false only happens if other.comment is null, otherwise it keeps going...

also, you started touching [common] and in the past I screwed up the mp3 code by applying automatic refactors like this.

I'm going to have to reject this commit, it's touching way too much.

Would love to keep the control issues you refactored on android.

Please try to keep the commits per project.

Reply:
That's ok, thanks for telling me.